### ISSUE TITLE
can't get any projects with Cloud tools for IntelliJ

### ISSUE THREAD

**Issue by victoy** [GoogleCloudPlatform/google-cloud-intellij#2221]
(Please ensure you are running the latest version of Cloud Tools for IntelliJ with _Help > Check for Updates_.)

- Cloud Tools for IntelliJ version: 18.6.1
- Google Cloud SDK version _(run `gcloud version` on the command line)_: Google Cloud SDK 208.0.2
- OS: MacOS High Sierra

**What did you do?**
While setting up cloud repository, I couldn't get my project list from IntelliJ GCP tools even though I got authentication properly.

**What did you expect to see?**
I would like to see my projects and choose it in IntelliJ. 

**What did you see instead?**
I'm using gcloud command on iterm(or terminal) instead of using IntelliJ GCP tools. 

(screenshots are helpful)
![screen shot 2018-07-17 at 12 08 29 pm](https://user-images.githubusercontent.com/3599924/42794509-27ed1020-89ba-11e8-824f-4e6bb3aaaa08.png)

_Feel free to deviate from this template as needed, especially if you are submitting a feature request._

**Comment by etanshaul** [comment]
hi @victoy thanks for the report.

just to make sure - you are signed in to the correct GCP account? In other words, when you go to the web console, log in as the same user, you can see your project list?

@ivanporty assuming @victoy does see projects via the web console, are there reasons why projects may be filtered out by our selector?

**Comment by ivanporty** [comment]
Hi @victoy, do you have any specifics for your projects by any chance - for example, you recently deleted one or re-created another project with the same name, or use a custom organization? What exactly "get my project list on gap" means (i.e. on gap). Thanks!

**Comment by victoy** [comment]
@etanshaul yes, I'm pretty sure that I used the proper one. I could see all my projects via web console and gcloud command with the same account.

@ivanporty I did rename the project. But... I could see my projects with "gcloud projects list" on terminal and web console after I changed the name.
* "get my project list on gap" ==> it's typo. (gap => gcp tool in IntelliJ)

**Comment by etanshaul** [comment]
@ivanporty if @victoy renamed the project to have the same name as a previously deleted one would this explain the issue? Is there a workaround you can recommend?

**Comment by victoy** [comment]
@ivanporty 
*if you add another project, does it appear in the project selector?
=> When I created new one, I could see it but still cannot see the renamed one.

*could you check details of your current project in the web console - any chance it's in "pending deleted" or some similar state?
=> It's not pending. The state looks same as new one.

**Comment by ivanporty** [comment]
@victoy Thank you for the information! It seems you can see your projects but the issue is with the renamed one. We will try to test and reproduce - in case you remember, you just renamed the project and that was it, it disappeared from the selector? Anything else specific to that rename - if you can give us approximate steps that'd be great.

**Comment by victoy** [comment]
1. Created new project. ( name: A)
2. Changed the project name to "B" ( A -> B)
3. I didn't realize it was pending at first. So, I created another one. ( name : B) I had two "B" projects. 
4. I realized that the first one has been changed successfully. So, I deleted new one (#3)

After that, I installed IntelliJ tool and tried to pull the projects. But I had a problem as I reported. I cannot say it disappeared. I couldn't see the project as soon as I installed the tool.
